app/src/pages/inside/logsPage/logsGrid/logMessageBlock/logMessageBlock.scss

```diff
@@ -0,0 +1,18 @@
+.log-message-block {
+  overflow: hidden;
+  color: $COLOR--charcoal-grey;
+  font-size: 12px;
+  font-family: Menlo, Monaco, Consolas, 'Courier New', monospace;
```

Comment:
is using of this font approved by UX Designer?

Reply:
Clarified it with designer. Added new font into `common/css/fonts`.